apps/openchallenges/db-cli/src/app.ts

```diff
@@ -25,95 +27,172 @@ export class App {
       .hook('preAction', () => this.setConfig(this.program.opts()));
 
     this.program
-      .command('seed')
+      .command('list-tables')
+      .description('list tables in a database')
+      .argument('<database>')
+      .action((database: string) => this.listTables(database))
+      .hook('preAction', () => this.setConfig(this.program.opts()));
+
+    this.program
+      .command('drop-tables')
+      .description('remove all tables from a database')
+      .argument('<database>')
+      .action((database: string) => this.removeTables(database))
+      .hook('preAction', () => this.setConfig(this.program.opts()));
+
+    this.program
+      .command('seed-database')
       .description(
-        'empty and seed the db with the JSON files from the directory specified'
+        'empty and seed a database with the JSON files from the directory specified'
       )
+      .argument('<database>')
       .argument('<directory>')
-      .action((directory: string) => this.seed(directory))
+      .action((database: string, dir: string) => this.seed(database, dir))
       .hook('preAction', () => this.setConfig(this.program.opts()));
 
     this.program
       .option('-d, --debug', 'output extra debugging')
-      .option(
-        '--uri <uri>',
-        'MongoDB uri',
-        'mongodb://localhost:27017/openchallenges'
+      .addOption(
+        new Option(
+          '--host <host>',
+          'IP address or DNS of the database server'
+        ).default(config.mariadb.host, 'value defined in config file')
+      )
+      .addOption(
+        new Option('--port <port>', 'database server port number').default(
+          config.mariadb.port,
+          'value defined in config file'
+        )
       )
-      .option('--username <username>', 'MongoDB username', 'openchallenges')
-      .option('--password <password>', 'MongoDB password', 'changeme');
+      .addOption(
+        new Option('--user <user>', 'user to access database').default(
+          config.mariadb.user,
+          'value defined in config file'
+        )
+      )
+      .addOption(
+        new Option('--password <password>', 'user password').default(
+          config.mariadb.pass,
+          'value defined in config file'
+        )
+      );
   }
 
   public async gracefulShutdown(msg: string, callback: any): Promise<void> {
     logger.debug('Gracefully shutdown');
-    if (this.mongoose) {
-      await this.mongoose.connection.close();
+    if (this.conn) {
+      await this.conn.end();
     }
     callback();
     return Promise.resolve();
   }
 
-  private async ping(): Promise<void> {
+  private async listDatabases(): Promise<void> {
     try {
-      this.mongoose = await connectToDatabase();
-      const pong = await pingDatabase();
-      logger.info(pong ? 'pong' : 'No pong received');
+      this.conn = await connect();
+      const databases = await listDatabases(this.conn);
+      console.log('database'.padEnd(this.padding, ' ') + '\ttables_count');
```

Comment:
Use the logger, not `console.log`. If the logger is configured to save logs to a file, then this content would not be captured.

Reply:
Ah okay, so use `logger` for all STDOUT?

Reply:
Using `logger`:
![Screen Shot 2023-02-16 at 10 40 52 AM](https://user-images.githubusercontent.com/9377970/219458007-a5a8ed67-fdcd-4481-9cbd-1b0823076a53.png)

Is this sort of output still useful? (pinging @rrchai for feedback as well)

Reply:
^ we could do something less visual/tabular, e.g.

![Screen Shot 2023-02-16 at 10 45 45 AM](https://user-images.githubusercontent.com/9377970/219458932-17bffa7c-2f59-48c8-8adc-33c78bf1ca0e.png)

Reply:
> use logger for all STDOUT?

Yes

I like the tabular format. If we already have it we may as well keep it.

Reply:
@vpchung the first format looks good to me!

Reply:
Okay, cool!  I will keep the tabular format.  The padding has been updated to be dynamic (was originally static).